packages/TelegramClient-Core.package/TCCChatsHandler.class/instance/addChat..st

```diff
@@ -4,12 +4,12 @@ addChat: aChatEvent
 	| chat | 
 	chat := TCCChat newFromChatEvent: aChatEvent.
 	(chat type = 'chatTypeSupergroup') ifTrue: [
-		self loadSuperGroupChatMemberCountFor: chat with: (((aChatEvent at: 'chat') at: 'type') at: 'supergroup_id').
+		self loadSuperGroupInfoFor: chat with: (((aChatEvent at: 'chat') at: 'type') at: 'supergroup_id').
 	].
 	(chat type = 'chatTypeBasicGroup') ifTrue: [
-		self loadBasicGroupChatMemberCountFor: chat with: (((aChatEvent at: 'chat') at: 'type') at: 'basic_group_id').
+		self loadBasicGroupInfoFor: chat with: (((aChatEvent at: 'chat') at: 'type') at: 'basic_group_id').
 	].
 
 	self app chats
-		add: chat; 
-		notify.
\ No newline at end of file
+			add: chat; 
+			notify.
```

Comment:
why indent like this? A single indentation should be just as readable, as the subject is on its own line